libs/orchestrator-ui-components/src/lib/components/Table/Table.tsx

```diff
@@ -0,0 +1,143 @@
+// Todo: issue on Eui's Github: https://github.com/elastic/eui/issues/6616
+import 'regenerator-runtime/runtime';
+
+import {
+    EuiDataGrid,
+    EuiDataGridCellValueElementProps,
+    EuiDataGridStyle,
+} from '@elastic/eui';
+import { useRef, useState } from 'react';
+import {
+    columnSortToEuiDataGridSorting,
+    ControlColumn,
+    DataSorting,
+    getInitialColumnOrder,
+    TableColumns,
+} from './columns';
+import {
+    EuiDataGridControlColumn,
+    EuiDataGridPaginationProps,
+} from '@elastic/eui/src/components/datagrid/data_grid_types';
+
+// Total height of grid button bar, table header and pagination bar
+const EUI_DATA_GRID_HEIGHT_OFFSET = 103;
```

Comment:
This is not a height that's relevant to the multiplier? (i remember you showed that in one of the first meetings about the project, but forgot the exact name)